support-frontend/assets/pages/[countryGroupId]/checkout/helpers/formDataExtractors.ts

```diff
@@ -14,21 +14,21 @@ export const extractPersonalDataFromForm = (
 	email: formData.get('email') as string,
 });
 
-type FormAddressFields = {
-	deliveryAddress?: {
-		lineOne: string;
-		lineTwo: string;
-		city: string;
-		state: string;
-		postCode: string;
+export type FormAddressFields = {
+	billingAddress: {
+		lineOne?: string | null;
+		lineTwo?: string | null;
+		city?: string | null;
+		state?: string | null;
+		postCode?: string | null;
 		country: IsoCountry;
 	};
-	billingAddress: {
-		lineOne?: string;
-		lineTwo?: string;
-		city?: string;
-		state: string;
-		postCode: string;
+	deliveryAddress?: {
+		lineOne?: string | null;
+		lineTwo?: string | null;
+		city?: string | null;
+		state?: string | null;
+		postCode?: string | null;
```

Comment:
I think if there is a delivery address then line one, city and postcode will always be present won't they?

Reply:
The changes here were to make these consistent with `RegularPaymentRequestAddress` defined here: https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/helpers/forms/paymentIntegrations/readerRevenueApis.ts#L145